### DISCUSSION TITLE
Any planned action on #8619?

### DISCUSSION THREAD

**Discussion by craigmcc** [react-hook-form/react-hook-form#8655] [General]
I am not allowed to reopen #8619 because of GitHub rules.  I added a CodeSandbox example of the compile errors in a comment.  Several other folks have indicated that they have observed the same situation.  Several other complaints about TypeScript errors on FieldErrors seem likely to have the same root cause.

This problem means I would completely break the build of any of my applications that I tried to update to use this build.  API breakage like that (in a dot.dot release, especially) is not a good sign.  I REALLY like the way RHF approaches this whole form handling area, but cannot afford to have my applications broken at build time.

Is there any plan to deal with this issue?

**Comment by bluebill1049** [comment]
https://github.com/react-hook-form/react-hook-form/issues/8653  when you are using `useFormContext` it's missing the generic, and causing the type checking failure and compile error. Tip: you should always provide generic when using context or its simply `any`

**Reply by craigmcc** [reply]
In my case, I am not using useFormContext -- just useForm.

**Reply by bluebill1049** [reply]
Then it's a probably different issue altogether. Until you can provide simple/reproducible codesandbox isolated to the issue, I won't have the time to go through your entire project. Even a failure type test can help. Alternatives:

- fork the repo
- stay at the version which works for you
- see if you can contribute and patch the potential root cause

**Reply by craigmcc** [reply]
https://codesandbox.io/s/goofy-fast-vdssvf

**Reply by bluebill1049** [reply]
I would appreciate it if the example is isolated from this one in the issue report rather your entire project with all the side dependencies https://github.com/react-hook-form/react-hook-form/issues/8653

**Reply by craigmcc** [reply]
8653 looks like it might be related, but is not my concern.  My issue (#8619) is illustrated by:

https://codesandbox.io/s/goofy-fast-vdssvf
